scanner/src/lib/dataproducts/Textile_Garment_ManufacturerInformation_v0.1.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  /*
+    Docs:
+
+    https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/Textile/Garment/ManufacturerInformation_v0.1.py
+
+    https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/Textile/Garment/ManufacturerInformation_v0.1
+
+    https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_Textile_Garment_ManufacturerInformation_v0_1_DigitalProductPassport_Textile_Garment_ManufacturerInformation_v0_1_post
+  */
+
+  import DataRow from "$components/DataRow/index.svelte"
+  import Divider from "$components/Divider/index.svelte"
+  import Article from "$components/Article/index.svelte"
+  import { localizeDate } from "$lib/common"
+
+  type Location = {
+    city?: string
+    country: string
+  }
+
+  type Manufacturer = {
+    facilityId?: string
+    manufacturerName: string
+    manufacturingDate: string
+    manufacturingLocation: Location
+    manufacturingPhase: string
+  }
+
+  export let data: {
+    manufacturers: Manufacturer[]
+  }
+
+  function formatLocation(location: Location): string {
+    if (location.city) {
+      return `${location.city}, ${location.country}`
+    }
+    return location.country
+  }
+</script>
+
+<Article>
+  {#if data.manufacturers.length > 0}
+    {#each data.manufacturers as m, index}
+      <div class="manufacturer">
+        <DataRow label="Manufacturer name" value={m.manufacturerName} />
+        <DataRow label="Facility ID" value={m.facilityId} />
+        <DataRow label="Manufacturing date" value={localizeDate(m.manufacturingDate)} />
+        <DataRow label="Manufacturing location" value={formatLocation(m.manufacturingLocation)} />
```

Comment:
Would be pretty nice if the country code could be converted to human readbale name ; i.e. instead of showing "EST" showing "Estonia".